canonical-web-and-design/ubuntu.com
# Clean up remaining /core/smartstart links and copy

## Done

- Updated all /core/smartstart links to /core/services
- Reinstated the guides, and updated the index guide according to the [copy doc](https://docs.google.com/document/d/1_RM806MdeRB6MDU8yoUyQFdZqopJobiKrqzgwUQm8J8/edit)
- Updated two instances of SMART START mentioned in copy to "IoT Professional Services"

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Visit https://ubuntu-com-11528.demos.haus/core/services/guide
- Check that it matches the [copy doc](https://docs.google.com/document/d/1_RM806MdeRB6MDU8yoUyQFdZqopJobiKrqzgwUQm8J8/edit)
- Make sure you can navigate through the guides and that the links within the copy to other docs work
